cardano-testnet/test/Test/ShutdownOnSlotSynced.hs

```diff
@@ -57,15 +58,24 @@ hprop_shutdownOnSlotSynced = H.integrationRetryWorkspace 2 "shutdown-on-slot-syn
     H.cat (nodeStdout node)
     H.cat (nodeStderr node)
   mExitCodeRunning === Right ExitSuccess
+
   logs <- H.readFile (nodeStdout node)
-  slotTip <- case find (isInfixOf "Closed db with immutable tip") (reverse (lines logs)) of
```

Comment:
This code is asserting that the node has (or will soon exit)

Is the new code asserting on the same thing?

Reply:
I updated the PR to look for `TraceOpenEvent.ClosedDB` as well.